components/brave_rewards/resources/ui/components/checkout/termsOfSale/index.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { Text } from './style'
+
+interface TermsOfSaleProps {
+  text: string
+}
+
+export function TermsOfSale (props: TermsOfSaleProps) {
```

Comment:
This component can be removed and just put the whole text in translations. Example of translation https://github.com/brave/brave-core/blob/a306c7e071f6f3935c32c5e88a8863d0d19badf7/components/resources/brave_components_strings.grd#L308. We do it like this so other languages can move link around as order of words is not the same in all languages.